.stitch/designs/login_desktop.html

```
@@ -0,0 +1,203 @@
+<div class="w-full max-w-[1100px] grid md:grid-cols-2 rounded-[2.5rem] overflow-hidden shadow-[0_32px_64px_-16px_rgba(48,41,80,0.12)] bg-surface-container-lowest relative z-10">
+<!-- Branding/Visual Side -->
+<div class="hidden md:flex flex-col justify-between p-12 kinetic-gradient text-on-primary">
+<div>
+<span class="text-2xl font-black tracking-tighter text-on-primary">SkillSwap</span>
+</div>
+<div class="space-y-6">
+<h1 class="text-5xl font-extrabold tracking-tight leading-[1.1]">The Kinetic Studio of Knowledge.</h1>
+<p class="text-on-primary/80 text-lg max-w-sm font-medium">Connect with mentors, swap skills, and grow your expertise in a collaborative fluid environment.</p>
+</div>
+<div class="flex -space-x-3">
+<img class="w-12 h-12 rounded-full border-4 border-primary object-cover" data-alt="portrait of a young smiling professional woman with bright studio lighting" src="https://lh3.googleusercontent.com/aida-public/AB6AXuA0vnMJjVWOBTDNfcSp_9_nPQmcFdjWurIS4vV8W1oHme-fWUMzyYkQFPlsBEj6-9aTqWVLp2eWS20fDz2PoLPqII43rF4MgE7lGMZvXEbHAccqLuEXtc9tNwIKeTRp26k9CJ4gau1_sIH8KJ9Kfz9Zzusi_cPG7N9Bo9P1XU-sBjqZVuv1ZXjd9jLDGmzvYINMDIhXTmLP8ei8WU8eEKfNyvQeH6jzhpU6Pc38VBfnDio1HL4TloiFISVPY7IkYG7oOTqDmlvagFg"/>
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)